udata/search/fields.py

```diff
@@ -56,7 +58,20 @@ def get_value_filter(self, value):
 
 
 class TermsFacet(Facet, DSLTermsFacet):
-    pass
+
+    def add_filter(self, filter_values):
+        if not filter_values:
```

Comment:
I think that given than the only case we want to handle is `OR`, it could be refactored as:
```python
def add_filter(self, filter_values:
    if OR_OPERATOR not in filter_values:
        return super(TermsFacet, self).add_filter(filter_values)
    values = filter_values.split(OR_OPERATOR)
    return any(super(TermsFacet, self).add_filter(v) for v in values)
```
And so it preserve the parent logic.
What do you think about ?